qa/rpc-tests/spentindex.py

```diff
@@ -29,7 +32,8 @@ def setup_chain(self):
     def setup_network(self):
         # -insightexplorer causes spentindex to be enabled (fSpentIndex = true)
 
-        self.nodes = start_nodes(3, self.options.tmpdir,
+        self.nodes = start_nodes(
+            3, self.options.tmpdir,
```

Comment:
I'd prefer that this `getblockhashes` PR didn't include all the noise from formatting changes that, if desired, should have been in #3953.

Reply:
Okay, I was just trying to get pyflakes to complain less, but I'll undo those changes.

Reply:
Agreed, however, I don't mind if these formatting changes are included in this PR. It might have made more sense as a separate commit to distinguish it from the other parts of this PR, but I would be ok with including this as is.

Reply:
Yep, I have no problem with the changes overall. The issue is that being so close to the RC, these kinds of changes just add noise to the PR and make it harder to review.